exasol_advanced_analytics_framework/udf_framework/query_handler_runner_udf.py

```diff
@@ -33,6 +31,32 @@
 from exasol_advanced_analytics_framework.udf_framework.udf_connection_lookup import UDFConnectionLookup
 
 
+def create_bucketfs_location_from_conn_object(bfs_conn_obj) -> bfs.path.PathLike:
+    bfs_params = json.loads(bfs_conn_obj.address)
+    bfs_params.update(json.loads(bfs_conn_obj.user))
+    bfs_params.update(json.loads(bfs_conn_obj.password))
+    return bfs.path.build_path(**bfs_params)
+
+
+def upload_via_joblib(location: bfs.path.PathLike, object: Any):
+    with NamedTemporaryFile() as temp_file:
+        joblib.dump(object, temp_file.name)
+        temp_file.flush()
+        temp_file.seek(0)
+        data = b''.join(temp_file)
+        # strangely location.write(temp_file) did not write any data to BFS
```

Comment:
I think it's because the NamedTemporaryFile is not seen as a file-like object.

Reply:
Ah, this sounds plausible, thanks!